src/components/kytos/tabs/tabs.vue

```diff
@@ -126,11 +145,20 @@ export default {
       // Get all elements with class="tab-nav" and remove the class "active"
       tablinks = document.getElementsByClassName("tab-nav");
       for (i = 0; i < tablinks.length; i++) {
-        tablinks[i].className = tablinks[i].className.replace(" active", "");
+        if (tablinks[i].className.includes(tabName)){
+          tablinks[i].className += " active"
+          this.lastTab = i
+        } else {
+          tablinks[i].className = tablinks[i].className.replace(" active", "");
+        }
       }
 
-      // TODO: Fix this active button
-      // Show the current tab, and add an "active" class to the button that opened the tab
+      /**TODO: Show the current tab by adding an "active" class
+       * to the button that opened the tab ->
+       * [Done - But maybe in the future change it to use
+       * event listeners if needed like:
+       * **evt.currentTarget.className += " active"**]
```

Comment:
If the TODO was Done, just remove it. Leave the "maybe in the future..." if you want.